tests/frontier/scenarios/common.py

```diff
@@ -190,20 +190,40 @@ def make_gas_hash_contract(pre: Alloc) -> Address:
     So that if we can't check exact value in expect section,
     we at least could spend unique gas amount.
     """
+    # EVM memory variables
+    variable_byte_offset = MemoryVariable(0)
+    variable_current_byte = MemoryVariable(32)
+
+    # Code for memory initialization
+    initialize_code = variable_byte_offset.store(0)
+    calldata_copy = Op.JUMPDEST + Op.CALLDATACOPY(
+        dest_offset=variable_current_byte.offset + 32 - 1,
```

Comment:
32, -1  magic numbers?

Reply:
It was 63 before, we were putting it in the last byte of the variable.
So the offset, plus its size (32), minus the one byte.